modules/snmp/snmp.go

```diff
@@ -157,5 +167,7 @@ func (s *SNMP) Collect() map[string]int64 {
 }
 
 func (s SNMP) Cleanup() {
```

Comment:
```suggestion
func (s *SNMP) Cleanup() {
```

Should we use pointer receiver in this method?